mono/mini/genmdesc.py

```diff
@@ -95,10 +95,20 @@ def parse_mini_ops (target_define):
     return opcodes
 
 def parse_input(infile, opcodes):
+
+    # Comments are pound sign to end of string.
+    remove_comments = re.compile ("#.*")
+
     for line in infile:
         line = line.strip ()
-        if line == "" or line.startswith ("#"):
+
```

Comment:
Extra line

Reply:
I have to insert lines here, might as well insert more and make it more readable.